lib/sub-process.ts

```diff
@@ -32,7 +32,18 @@ export function execute(command, args, options): Promise<string> {
 
     proc.on('close', (code) => {
       if (code !== 0) {
-        return reject(new Error(stderr || stdout));
+        debug(`Child process failed with code: ${code}`);
+        debug(`
+          ----------------
+          STDERR:
+          ${stderr}
+        `);
```

Comment:
this will look a bit funny with leading spaces?

Reply:
Ah, you're right.
I've changed our debug function slightly so that it can take multiple strings and log them on new lines.